src/Microsoft.PowerShell.Commands.Utility/Microsoft.PowerShell.Commands.Utility.csproj

```diff
@@ -33,12 +33,10 @@
     <Compile Remove="commands\utility\ShowCommand\ShowCommandParameterSetInfo.cs" />
```

Comment:
@SteveL-MSFT, your last commit had 2 failures in `PowerShell-CI-linux`
Verify approved aliases list.All approved Cmdlets present (no new Cmdlets added, no Cmdlets removed)
```powershell
Expected $null or empty, but got Update-List.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 584
584:         $result | Should -BeNullOrEmpty
```
Verify approved aliases list.All present Cmdlets should have the correct ConfirmImpact
```powershell
Expected $null or empty, but got @(@{Name=Write-Warning; ConfirmImpact=None; SideIndicator=<=}).
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/DefaultCommands.Tests.ps1: line 606
606:         $result | Should -BeNullOrEmpty
```